components/Template/TemplateDesigner/LayoutElement.vue

```diff
@@ -276,18 +282,12 @@ export default {
             this.newHeight = height;
         },
         initElementStyleForResizeState() {
-            this.$el.style.position = 'absolute';
             this.$el.style.width = `${this.startWidth}px`;
             this.$el.style.height = `${this.startHeight}px`;
-            this.$el.style.border = '2px solid #00bc87';
-            this.$el.style.zIndex = '5';
         },
         resetElementStyleForEndResizeInteraction() {
```

Comment:
Better is use CSS class with property:
`width: 'unset;
height: 'unset''`

Reply:
CSS class won't working - inline styles will overwrite it